README.md

```diff
@@ -68,134 +62,58 @@ then as soon as your PR is merged into master, it will be merged into the curren
 ### Creating a version
 
 Here are the steps necessary to create a new documentation version. Let's assume the current
-version of Istio is 1.3 and you wish to introduce 1.4 which has been under development.
+version of Istio is 1.6 and you wish to introduce 1.7 which has been under development.
```

Comment:
Does `creating a version` imply major/minor or does it also include patch releases?